vg.yaml

```diff
@@ -2,17 +2,18 @@ apiVersion: volumegroup.storage.dell.com/v1
 kind: DellCsiVolumeGroupSnapshot
 metadata:
   # Name must be 13 characters or less in length
-  name: "vg-snaprun1"
+  name: "vg-snaprun52"
```

Comment:
what is the reason name change?

Reply:
just a name change 
do you want me to change this ?

Reply:
reverted it back